test/es6/rlexe.xml

```diff
@@ -95,13 +95,15 @@
       <tags>BugFix</tags>
     </default>
   </test>
+  <!-- Disabling for now because the test is flaky and fails in some situations
```

Comment:
Uh yeah, actually. Thanks for reminding me to go look it up. We made this change in another version of the source a while back when the issue initially came up. There's a bug tracking it. Also it's definitely a compile bug and not just test flakiness.